go/vt/vttablet/tabletserver/state_manager.go

```diff
@@ -531,23 +542,25 @@ func (sm *stateManager) unserveCommon() {
 	log.Infof("Finished execution of handleShutdownGracePeriod")
 	defer cancel()
 
-	log.Infof("Started online ddl executor close")
+	log.Info("Started system health monitor close")
+	sm.systemHealth.Close()
+	log.Info("Finished system health monitor close. Started online ddl executor close")
 	sm.ddle.Close()
-	log.Infof("Finished online ddl executor close. Started table garbage collector close")
+	log.Info("Finished online ddl executor close. Started table garbage collector close")
 	sm.tableGC.Close()
-	log.Infof("Finished table garbage collector close. Started lag throttler close")
+	log.Info("Finished table garbage collector close. Started lag throttler close")
 	sm.throttler.Close()
```

Comment:
```go
	sm.systemHealth.Open()
	sm.tableGC.Open()
	sm.ddle.Open()
```
We follow the order of Open and Close.

as open happened before  `tableGC` and `ddle`, the close should happen after `tablegc` and `ddle`